src/client/app/containers/groups/EditGroupContainer.js

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
This file might be wrong because of a bad merge, check the right version in groupUI.